CanHazFunny/CanHazFunny/JokeService.cs

```diff
@@ -2,14 +2,23 @@
 
 namespace CanHazFunny
 {
-    public class JokeService
+    public class JokeService : IJokeService
     {
         private HttpClient HttpClient { get; } = new();
 
         public string GetJoke()
         {
-            string joke = HttpClient.GetStringAsync("https://geek-jokes.sameerkumar.website/api").Result;
-            return joke;
+            Uri uri = new Uri("https://geek-jokes.sameerkumar.website/api?format=json%22");
```

Comment:
```suggestion
            Uri uri = new Uri("https://geek-jokes.sameerkumar.website/api?format=json");
```
Don't include this double quote in your URL, you aren't actually getting the json format back.